shell/packages/sandstorm-db/db.js

```diff
@@ -57,6 +57,7 @@ if (Meteor.isServer && process.env.LOG_MONGO_QUERIES) {
 //                an identicon will be used.
 //       pronoun: One of "male", "female", "neutral", or "robot".
 //   unverifiedEmail: If present, a string containing an email address specified by the user.
+//   referredBy: The Account that referred this Identity.
```

Comment:
Fixed in latest push (I said `ID of the Account` not `The ID of the account`; I figure Account is basically as much of a proper noun as User even though it's not quite).